app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class Admin::UsersController < ApplicationController
+  def index; end
+
+  def new; end
+
+  def create
+    user = User.create(user_params.merge(password: Devise.friendly_token[0, 20]))
```

Comment:
`password: Devise.friendly_token[0, 20])` Whats happening here?  Are we generating a password for the user?

Reply:
Yes, we can't save the user without a password so I'm generating a default one. It's a similar process as we do in `User` `from_omniauth` method.